frontend/src/ui/ArticleContentComponent.tsx

```diff
@@ -24,11 +30,12 @@ export const ArticleContentComponent: FC<ArticleContentProps> = ({
   }, []);
 
   return (
-    <div className="flex flex-col items-start py-5 space-x-8 space-y-4">
-      <div className={"pb-3 text-2xl"}>{title}</div>
-      {/*  TODO add info on publication time*/}
+    <div className="flex flex-col  py-5 space-y-4 bg-background-dark lg:p-5 rounded-lg">
+      <div className={"text-2xl"}>{title}</div>
```

Comment:
```suggestion
    <div className="flex flex-1 flex-col p-4 lg:p-6 space-y-4 bg-gradient-to-b from-background-dark rounded-lg">
      <Text h2>{title}</Text>
```
tak bym zrobił - trochę lżej i spójniej z listą, możesz obczaić czy ci pasuje